### DISCUSSION TITLE
NFR - Version Support

### DISCUSSION THREAD

**Discussion by DanielaWuensch** [eclipse-edc/Connector#1673] [Ideas]
# Feature Request
Non-functional Requirement -  Version Support
Version Planning, Reporting and Support - Report version information to enable version reporting in any central landscape management tool 
versions need also to be available in branches in a stable way
version plan and support with bug fixing 
document what is the versioning strategy - how long will a version supported and how / if / that different versions can interact with each other

## Which Areas Would Be Affected?
all, including DPF, CI, build, transfer, etc._

## Why Is the Feature Desired?
Operation Requirement - Commitment for a stable 1.0.0 needed to enable productive usage

## Solution Proposal
usage of prepared statements

## Type of Issue
non-functional requirement

## Checklist
Version plan documented and given as guidance

**Comment by zeier** [comment]
See also decision record [Automating the EDC release process](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/tree/main/docs/developer/decision-records/2022-07-06-release-automation).
